test/e2e/framework/deployment.go

```diff
@@ -354,6 +357,11 @@ func (f *Framework) NewHttpbinDeployment() {
 	f.NewDeployment(HTTPBinService, "k8s.gcr.io/ingress-nginx/e2e-test-httpbin@sha256:c6372ef57a775b95f18e19d4c735a9819f2e7bb4641e5e3f27287d831dfeb7e8", 80, 1)
 }
 
+// NewMemcachedDeployment creates a new single replica deployment of memcached
+func (f *Framework) NewMemcachedDeployment() {
+	f.NewDeployment(MemcachedService, "docker.io/memcached@sha256:944f6eb53be39dfcb895ab110d685e4f0320ffa00b8a1c0fe35aeb52f4f89d30")
```

Comment:
@aledbf should we push memcached image to `k8s.gcr.io/ingress-nginx` instead of relying `docker.io`?

Reply:
I don't think that is a good idea to start using the `k8s.gcr.io/ingress-nginx` for dependencies like this one, or redis for instance.